src/components/dialogs/Dialog.js

```diff
@@ -0,0 +1,127 @@
+import Toggleable from '../../mixins/toggleable'
```

Comment:
We talked about the mixin for card-actions. If possible, I'd prefer to use that as we can ensure the behavior is the same across different components. https://github.com/vuetifyjs/vuetify/blob/feature/pickers/src/mixins/card-actions.js